tests/sources/xml/test_datacite.py

```diff
@@ -609,7 +609,7 @@ def test_get_file_formats_transforms_correctly_if_fields_missing():
 
 
 def test_get_format_success():
-    assert Datacite.get_format() == "electronic resource"
+    assert Datacite.get_format({}) == "electronic resource"
```

Comment:
Hmm, rather than passing in an empty dictionary, can we set a default of `None` to `_source_record` parameter for the field methods that don't use source records? 🤔 Would change the function signature very slightly:

```
def get_format(cls, _source_record: Tag | None = None) -> str:
        return "electronic resource"
```

```

Reply:
Very good call, much cleaner, thanks!